addons/mail/static/tests/mock_server/mock_models/mail_message.js

```diff
@@ -384,12 +390,11 @@ export class MailMessage extends models.ServerModel {
         if (reactions.length === 0) {
             reaction_group = [["DELETE", { message: this.browse(id), content: content }]];
         }
+        store.add(this.browse(id), { reactions: reaction_group });
         BusBus._sendone(
             this._bus_notification_target(id),
             "mail.record/insert",
-            new mailDataHelpers.Store(this.browse(id), {
-                reactions: reaction_group,
-            }).get_result()
+            store.get_result()
```

Comment:
This should be the same as python, a different Store.
Also it would be nice to introduce the missing methods, at least `_bus_send_reaction_group` and `_reaction_group_to_store`. (not need for `_bus_send_store` in this PR)

Reply:
Done